tools/interactive/interactivetool_jupyter_notebook.xml

```diff
@@ -9,18 +9,11 @@
         </entry_point>
     </entry_points>
     <environment_variables>
-        <environment_variable name="HISTORY_ID" strip="True">${__app__.security.encode_id($jupyter_notebook.history_id)}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
-        <environment_variable name="REMOTE_HOST">${__app__.config.galaxy_infrastructure_url}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="HISTORY_ID">$__history_id</environment_variable>
+        <environment_variable name="REMOTE_HOST">$__galaxy_url</environment_variable>
         <environment_variable name="GALAXY_WEB_PORT">8080</environment_variable>
```

Comment:
what about a `galaxy_infrastructure_web_port`?

Reply:
That was removed in dev I think (with https://github.com/galaxyproject/galaxy/commit/e50807a7c6ce050cedab623546f43d1b2b75ac52) - seems like we decided it doesn't do anything most of the time? Is GALAXY_WEB_PORT actually used by the container?